ADconnection.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bas
```

Comment:
The shebang line `#!/bin/bas` seems to be incorrect. It should likely be `#!/bin/bash` to specify the correct path to the bash interpreter.